browser/css/jssidebar.css

```diff
@@ -165,7 +165,9 @@ div#thousandseparator.checkbutton.jsdialog.sidebar {
 #document-container ~ #sidebar-dock-wrapper .sidebar.jsdialog.vertical > .sidebar.jsdialog.row > table.sidebar.ui-grid > tr.sidebar > td.sidebar:last-child:not(:first-child),
 .sidebar.jsdialog.cell > #table-box3 > .sidebar.jsdialog.row > .sidebar.jsdialog.cell:last-child,
 .sidebar.jsdialog.cell > #table-box4 > .sidebar.jsdialog.row > .sidebar.jsdialog.cell:last-child,
-.sidebar.jsdialog.cell > #table-box4 > .sidebar.jsdialog.row > .sidebar.jsdialog.cell:nth-child(4) {
+.sidebar.jsdialog.cell > #table-box4 > .sidebar.jsdialog.row > .sidebar.jsdialog.cell:nth-child(4),
+.sidebar.cell:not(:nth-child(1)) > .vertical > .row > .toolbox,
+.sidebar.cell:not(:nth-child(1)) > .toolbox {
```

Comment:
Thanks but I'm afraid these will affect other cases. Ideally we would avoid those multiple :nth-child, I also wonder if those #table-box ids can be redone (in core) :thinking: 

TLDR: Do you think it will be possible to target those via other classes or maybe better straight from JS?

Reply:
yes you are right it's complex but we have in the sidebar sometimes table and sometimes div's so it's very difficult to catch dem.

Here the idea is to catch the right element which mean the last element within a table, div, ... excl. if there is only one element.

I also want to remove all the other complex stuff, but reduction (delte) stuff mean a lot of review and checks. So first add one css rule to fix the existing "visual bug" and than reduce stuff.

Reply:
I'm way further by reduce rules, but one step after another.